debian.sh

```diff
@@ -61,54 +72,63 @@ setup_network_config() {
   fi
 }
 
-# generate_mdadmconf "NIL"
+# generate_config_mdadm "NIL"
 generate_config_mdadm() {
   if [ -n "$1" ]; then
     MDADMCONF="/etc/mdadm/mdadm.conf"
-#    echo "DEVICES /dev/[hs]d*" > $FOLD/hdd$MDADMCONF
-#    execute_chroot_command "mdadm --detail --scan | sed -e 's/metadata=00.90/metadata=0.90/g' >> $MDADMCONF"; declare -i EXITCODE=$?
+    # echo "DEVICES /dev/[hs]d*" > $FOLD/hdd$MDADMCONF
+    # execute_chroot_command "mdadm --detail --scan | sed -e 's/metadata=00.90/metadata=0.90/g' >> $MDADMCONF"; declare -i EXITCODE=$?
```

Comment:
at this point it is just about shellchecking, changing functionality has been avoided on purpose.